main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_dict = json.load(players_file).items()
+
+    for player, player_attr in players_dict:
+        Race.objects.get_or_create(
+            name=player_attr["race"]["name"],
+            description=player_attr["race"]["description"],
+        )
+
+        for skill in player_attr["race"]["skills"]:
+            race = Race.objects.get(name=player_attr["race"]["name"])
+            Skill.objects.get_or_create(
+                name=skill["name"], bonus=skill["bonus"], race=race
```

Comment:
use single style of getting values by keys
use get() like u used here
                name=player_attr.get("guild")["name"],